### ISSUE TITLE
[BUG] bug when server has more than 1000 player

### ISSUE THREAD

**Issue by Pedro-Lucas14** [citizenfx/txAdmin#661]
Scope
Web

Feature Description
bug when server has more than 1000 player

![Screenshot_2](https://user-images.githubusercontent.com/84646506/184465093-7eadb710-3b93-4243-9cf6-f7a70b0c6a65.png)

Thank you

Use Case
No response

Proposed Solution

increase the limit that appears to 2048 the fivem key supports

Additional Info
No respons

**Comment by tabarra** [comment]
Sorry, what's the bug?  
Only issue I see there is that the left bar is cutting off the 1 in 1000 so it looks 000.  
  
Btw join our discord and ping me in #offtopic to get the top servers role and access to priority support.

**Comment by PC1up** [comment]
Please, for the love of god, use dark mode.
Love, Paul

**Comment by tabarra** [comment]
Fixed in v4.18.0.  
Now the size is dynamic.  
```js
const clientsToMargin = (maxClients) => {
    //each char is about 5px
    return 12 + (maxClients.toString().length * 5);
};
```
![image](https://user-images.githubusercontent.com/1808295/189490461-b45227c1-b7d6-4db5-9881-897f0dd4cce3.png)
